### DISCUSSION TITLE
Replacement of an old type of the HSR display

### DISCUSSION THREAD

**Discussion by akinobu1998** [HumanSupportRobot/QandA#8] [General]
Our HSR has an old version of the display and we are planning for a replacement. 
We followed the hsrb maintenance manual for the troubleshooting, and we found that the backlight of the display is broken. According to the manual, it seems we need to replace the display to the new version with 5V power supply cable. 
Could you provide us with the specifications (i.e. model number of the display) of the display?

**Comment by hsr-admin** [comment]
Please email us at the address below so we can send you a replacement DISPLAY.
hsr-support@robocup.or.jp
